docs/en-US/component/form.md

```diff
@@ -111,77 +111,81 @@ form/size-control
+| `validate` | triggers after a form item is validated | `(prop: FormItemProp, isValid: boolean, message: string) => void` |
+
+### Form Slots
+
+| Name | Description               | Subtags  |
+| ---- | ------------------------- | -------- |
+| -    | customize default content | FormItem |
+
+## Form Item API
+
+### Form Item Attributes
+
+| Attribute        | Description                                                                                                                                                   | Type                              | Default     |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------- | ----------- |
+| `prop`           | A key of `model`. It could be an array of property paths (e.g `['a', 'b', 0]`). In the use of `validate` and `resetFields` method, the attribute is required. | `string \| string[]`              | —           |
+| `label`          | Label text.                                                                                                                                                   | `string`                          | —           |
+| `label-width`    | Width of label, e.g. `'50px'`. `'auto'` is supported.                                                                                                         | `string \| number`                | —           |
+| `required`       | Whether the field is required or not, will be determined by validation rules if omitted.                                                                      | `boolean`                         | `false`     |
+| `rules`          | Validation rules of form, see the [following table](#formitemrule), more advanced usage at [async-validator](https://github.com/yiminghe/async-validator).    | `FormItemRule \| FormItemRule[]`  | —           |
+| `error`          | Field error message, set its value and the field will validate error and show this message immediately.                                                       | `string`                          | —           |
+| `show-message`   | Whether to show the error message.                                                                                                                            | `boolean`                         | `true`      |
+| `inline-message` | Inline style validate message.                                                                                                                                | `boolean`                         | `false`     |
+| `size`           | Control the size of components in this form-item.                                                                                                             | `'large' \| 'default' \| 'small'` | `'default'` |
+
+#### FormItemRule
```

Comment:
```suggestion
#### FormItem Rule
```

Reply:
This is TS type name

Reply:
Since this was written as a title, I think we should respect the the way how title was treated in the documentation, BTW, we need to annotate what `arrayable` is for the audience because they don't know about the actual type because it was not a built-in type.

Reply:
<img width="1201" alt="image" src="https://user-images.githubusercontent.com/6481596/156965035-1e17bc2e-b397-4287-9f7e-77cdd407ee40.png">

I think it needs to be consistent with the type in the table. And this is a level 4 heading that will not show up in the TOC.

Do you have a better way to label Arrayable?